navapbc/wic-participant-recertification-portal
# [PRP-180] Create initial setup for staff portal

## Ticket

https://wicmtdp.atlassian.net/browse/PRP-180

## Changes
> What was added, updated, or removed in this PR.

- Set up docker
- Set up formatting
- Set up initial directories

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

This PR creates the initial project structure for the staff portal, including the initial npm packages we will need, the initial project directories, and linting.

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

1. Start docker
2. Run `docker compose up -d --build`
3. The images should build successfully and all three containers should start up. Test by running `docker compose ps` and confirming none have an "Exited" status
4. Verify that the dev container is running at localhost:3033
5. Verify that the prod container is running at localhost:3333
6. When done, run `docker compose down` to stop and remove all containers
